source/shared/net/guides/run-node-windows.rst

```diff
@@ -159,7 +161,17 @@ When nodes are stopped or started, this is also recorded in the system event log
    .. image:: ../images/run-node/Node-setup-win-7.png
          :width: 50%
 
-If a node fails to start, for example because of a configuration issue, or stops unexpectedly, there will be an Error-level event in the Event Viewer that explains why.
+Node troubleshooting
+--------------------
+
+If a node fails to start, for example because of a configuration issue, or stops unexpectedly, there will be an Error-level event in the **Event Viewer** that explains why. The following are common error scenarios and possible solutions.
+
+- The nodes doesn't restart after setting the credentials. This error can occur when the specified location of the credentials file is incorrect. In the **Event Viewer** an error such as the following might appear. When this happens, change the path in the configuration file.
+
+   .. image:: ../images/run-node/Node-setup-win-10.png
+         :width: 60%
+
+- The node doesn't start baking, and in the log file you see the message *Baker keys are incorrect* repeatedly. It takes 1-2 hours after baker keys have been added to an account for the keys to be eligible for baking. If this problem persists beyond two hours, then the keys likely are incorrect. Double check that you are not using keys from testnet on mainnet, or vice-versa.
```

Comment:
These pertain specifically to when you set the baker credentials, so it may be better to put them in the guide on becoming a baker. Otherwise, it should be clarified here that these scenarios pertain to the situation where the user has set baker credentials in their configuration file. (In particular, in the context of this page, it is probably not clear what "setting the credentials" means> Also, if they haven't configured the baker then the node isn't going to start baking, but the second solution might be confusing without further context.)